shared/protocol/src/main/java/io/pravega/shared/protocol/netty/CommandEncoder.java

```diff
@@ -139,6 +139,11 @@ protected void encode(ChannelHandlerContext ctx, Object msg, ByteBuf out) throws
         } else if (msg instanceof WireCommand) {
             breakFromAppend(out);
```

Comment:
The `KeepAlive` message would cause the invocation of `breakFromAppend(out)` which is similar to the what is being done with new msg `Flush`.
Could you please describe the advantage of creating a new `WireCommand v/s using `KeepAlive`

Reply:
This new wireCommand flush ; will flush the pending writes ; but it does not write any additional messages to wire .. This means no keep alive to wire during flush.. This reduces the keepalives to segment store.